neo/UIntBase.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Neo
 {
-    
```

Comment:
Maybe remove this extra space?

Reply:
I used the tool to ident it, maybe this is the correct way of format this document?

Reply:
The other thread resolves this question... in fact I disabled this rule. We can enable it on the future, to prevent allowing people to putting spaces after braces. First step is having minimum compliance, then we get more rigid about it ;)